src/main/java/roomescape/theme/controller/ThemeApiController.java

```diff
@@ -0,0 +1,48 @@
+package roomescape.theme.controller;
+
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.theme.dto.BestThemeResponse;
+import roomescape.theme.dto.ThemeRequest;
+import roomescape.theme.dto.ThemeResponse;
+import roomescape.theme.service.ThemeService;
+
+@RestController
+@RequestMapping("/themes")
+public class ThemeApiController {
+
+    private final ThemeService themeService;
+
+    public ThemeApiController(ThemeService themeService) {
+        this.themeService = themeService;
+    }
+
+    @PostMapping
+    public ResponseEntity<ThemeResponse> add(@RequestBody ThemeRequest themeRequest) {
+        return ResponseEntity.status(HttpStatus.CREATED).body(themeService.add(themeRequest));
+    }
+
+    @GetMapping
+    public ResponseEntity<List<ThemeResponse>> findAll() {
+        return ResponseEntity.ok(themeService.findAll());
+    }
+
+    @GetMapping("/best")
+    public ResponseEntity<List<BestThemeResponse>> findBest() {
+        return ResponseEntity.ok(themeService.findBest());
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity<Void> deleteById(@PathVariable("id") Long id) {
+        themeService.deleteById(id);
+        return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
+    }
```

Comment:
```suggestion
    @DeleteMapping("/{id}")
    public ResponseEntity<Void> deleteById(@PathVariable("id") Long id) {
        themeService.deleteById(id);
        return ResponseEntity.noContent().build();
    }
```
다른 작성된 형식과 맞추면 좋을 것 같아요

Reply:
오 네, 해당 부분 리팩토링에 반영했습니다!